### ISSUE TITLE
Dateiupload

### ISSUE THREAD

**Issue by Mavv3006** [Mavv3006/mathbase#46]

**Comment by luisafuss** [comment]
Was genau ist damit gemeint? Dass die Bilder, die man Aufgaben hinzufügt auch gespeichert werden?

**Comment by Mavv3006** [comment]
> Was genau ist damit gemeint? Dass die Bilder, die man Aufgaben hinzufügt auch gespeichert werden?

Ja. Aktuell funktioniert der Button noch nicht und das sollte sich ändern.
